InnerEye/ML/SSL/byol/byol_moving_average.py

```diff
@@ -0,0 +1,64 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+
+import math
+
+import pytorch_lightning as pl
+import torch
+from pytorch_lightning import Callback
+
+
+class BYOLMAWeightUpdate(Callback):
```

Comment:
Too many capital characters for my liking. .Net coding guidelines, for example, suggest to have abbreviations with 3 or more characters camel cased, so it would be Byol here. Also, MA is not a well known abbrevation. ByolMovingAverageWeightUpdate?

Reply:
updated